source/docs/software/vision-processing/raspberry-pi/diagrams/vision-code-on-a-coprocessor.drawio.svg

```diff
@@ -0,0 +1,3 @@
ZNX8JhowYSoOOPicvbmecLGfRe1wzfQxWoF6A6uhrtHZoJz7ZDl+YPPGtfwTttTXsiraxbeWULOPG9hNHPIPW9KJgSc03/SufEoiYmcZUUhFe7xux3fP3VxS6eDBRGSVC48Ob393lb1k5OSBX1LxVByelb5YDRstgVqiwhjOMBCzbw7tMPGh0zzCT/3V1axx2ExG5Pppj+69wfD+g3CUVVcGvPUUKm5ka8hdL/fY5sIOtnUvIpDBRKovXUJnqnP8YejWgySFV0cWSKyM1MtpwSYWB12rVTS2w4MKRo9c3xcLOJ7A4gfepKEXt9Gpq9ygvvsYTHL3e5GdJ8BS54/eXN2wwqbv1+iilsE70EwQiWu+VnBPFThRkCFpiTf50chQhChPPAz7O2vli8+/QR1/f2cQx7MAOpz6wiOiP6ZPdcqosOWtbKa/lKon9qb8qUTZIj7SW/9M8AfzKVpfLTczSUGA2TJEl2SG7GYXCKorO5srTUJJuVJT/DdRRd/zYzXxRorgWvLrrl/ME9VfWv9G5dvBmXyIRHEaNwX6gBXe0pFywxZQ7QxyItNViEnv4U8jZ56DkRT0hXBrohKEFeSOn6RAwjnROhER4+Lv+hJ5FjFKssMg+vg7gCXo+wgt/EbxgLFAXlEOFXLlG8Jt1Pbolvmij29Hhkh4mjEDjUE9GjyPm6zNcbXmkSzLs/FkGgxnRQGAyDbsLYn3ruz76GvfiI3Tx/iOlr7Uo+C0cywJujDuc0eLlG87u+aiALNrXj+jdODGtPEM942ujj7rZxaLdjfOV2PXZsthzYWssJO9Y/57QxDk7BPhY8rvLKTuDCTkIAKNW+/TE7TGGd/xK+Jica6nY37mr9oyGMrFPb8uJHLKEpWWzh342bGmvI1ds8rQ6peVRS++b63GiMP4fl9vGJy9+v1vOoUm4YUYPUFnFomt4uaHtX37CjP5u6Ksxa2pSHwolMF4nxp1oU9EqLg0VkDefJZZqd8+XfCzeiyxWPRjQmwSq7ulX/VQwJ/qWboYFI04myGdMZsjsaC4pY+D6H/fkSnjdDv6JUNkhjUMH4OfLrWZwTFEYnsxoZdkU/uPqtPGzQ3lwvldRqrRfReLsgdVbav1Ul9FuRU/jbaXNvXNGNPgEJGwaNrtscv9NPnbDTFLfaIwhz/GLPLSR9B3BjD+rrj6NEvXYkKQqtNv59HtYYPeCefM69GSxNYiYqqxKeOlTMzpybkFvRD+zfxka8VNoSrzHqc4skD4FEfqj6Ria9ReVZENC3CmvSwuo7l38nDX02uOX3nqqH1wcNMi+OU8MC/wAGTFTaG+X6AJO/JSV8B+CAj8aiGy/bpkVmlVMs9/BdBsITFgLbC+dbJTHC++FjOtFrUEccV6fxC4wAjOy8g66kZQWVWF0aKB4jv5WthPTn1gTo4iUN+IPNBY2tcbyob+euLYLhvNnMsGsrPPp81BPZEei6QVXmwhY05JiD+RaOsOjZFck350+aV7+uJGNKifvxu2w63ZfP2GZKRdX7ZFUDD2i4lcjtiSz3z0EgZfFSo3DEjP6omvXUqXvyNJaEEGREBDlAIdhIX1UzqG1H3/upM9b//GIyLam2nTR2lqnnAMa1dxGTotd7XR0XXtZoGp4yFHTBpdqqwtqGzjj61KCIfzGzioB55WZ7lpZOP2mIWjjCuzjfQZvgkzmjgiZib/FMuHBYMRNTF6crnGEH0pWsxkqIf61XrUbeX5YlCptRnqQ8bCad84yB6n6Eg6i7e+X7XnCYnYr8dyZ69XhBiY72Vav76/yvFujenaGylmtJ1SnV61xaaZwNXH5+Fw/31DLcVuOSWwOe2vYHyghcchToTYjUzoYQ4IZ3TNuLK8xGTSKk56+Z0ynsum5u0EX1BEE91Bu5N8/IjI3vFuxfaU5kWP2EfjTtMh1RCe7qG7xVSRLnDSatcyK7y2N1mYhfsSFe1R15MuSDZEqxy2vZFYVMevSq1SjmADPrn9QoqHfCg9Q1e+3HoFL35zARzr2MsyAA55fcKz5NIW4lgp7S5QMC2GHYa8QA12D3IB9rGbtKxzIhC8msxitA2aduys9avDFw5wlv6vZxxwjRUInY5/dMj6zkh8ki3oWc9W8s5ztrW7wDG6lftghsRPzHEusblzfF3RTgp1m5O1wJyRakd2zFcsJoSr9Ob6N/L2PZZZ2OmReEfmqZ4Nh1AQeunXfLgcKxyjHNYb0Tt99P0SxhO+7mi6GO18vyksZ4qQJ0rWIvop8OUbql5M7iZoc9KikYb+LN4Rpr2SyJC+vREjHSdYpfFZ+ZfOPgx+8bGlFmtMOpNbPNF1z1pFy96zACvvQOGAb+/T90o9vvALi8Q6zYmiW2K6aq3XYEBLfhmax3Us5e3zY+glqI7NNuZ4Ht6X9bF3Hm66nzEJYN9JU2Qfs1tM5oHbChbFPoLd6rh9XvX74iydv1qswT4fF0+U3idFcEuT3UeiGZd9nDeSQ417gesyxEI/GUWXSn4XYXmma/a20KZ2VKJqDBFla4rcD9A6ETYsHQh9eH+ZUTcHWudlVp8Jyhowvmqx6Zd4OMM9277NlFLGX9fG8qxBkplir2Z+BGaaUAUYL+bmN0Eq1B4k7evX06pwW2Bsel3I1jZYg3fwumbr7Gk3agoCaqvKqvZg/FFTi9HQD2RE0JrZPRX24zQgrnBjlXj2uuqKWeU9UMTU7b5z70li5E+ceHquTZaOMapvfTn12vpQpd86o0pohExX2HshSXAkdwbdmVSYN7rTNxqWlDhfmgGLD1i7+akSmQVS51VKChJ1paQ+kGxCfgWZvz7OFy4erkVT/NXN3y6OL8YdYqJox1TXKKytkHVL66yfeTafWk33wCf51X6o19d3B8XxHtX17CI/tw/5P837azY7O0F6RgO/YfBvsiyn4QbiGkFAzYU4woJFL224trfPBln8ufEQ55WyNzbY963nm4CeO3Nbby75LElWkBlrv4NHLWhA2rEsTqWtX6Jn1fFloZstG+iU
```

Comment:
Please replace FMS with "field" or "field network".

Reply:
Does this look good?
![image](https://user-images.githubusercontent.com/15172054/82280953-f5ddc880-9944-11ea-82e7-2a3ba611c907.png)